### PR TITLE
use case example for use of api endpoints

### PR DESCRIPTION
**Pull request by hanne-opseth-rygg-4ss**

### CODE REVIEW

File: docs/user_guide/example.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
+Use case example
+================
+
+Here are two simple example on how you can use the API endpoints to access data. The first example demonstrates how to find all instruments associated with a field, and the second example demonstrates how to find all risers with a specified length and sort them by field.
```
Comment (branislav-jenco-4ss) [inline docs/user_guide/example.rst:4]: Here are two simple example**s** **showing** how you can use the API endpoints to access data.

File: docs/user_guide/example.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+To build your own application using your desired API endpoints, you can find a complete overview of the available API endpoints in the `4insight REST API`_ documentation.
+
+.. caution::
+    Do not utilize the INTERNAL API endpoints. You can however use the endpoints from version 1.0, 1.1 and 1.2.
```
Comment (branislav-jenco-4ss) [inline docs/user_guide/example.rst:9]: Nitpick, but not reason to use "utilize" when "use" is perfectly good :)

**Review by branislav-jenco-4ss — DISMISSED** [review]
Some nitpicks, otherwise lgtm!

**Comment by hanne-opseth-rygg-4ss** [comment]
Thanks for the feedback, fixed it now, you  need to approve once more. ,T

**Review by branislav-jenco-4ss — APPROVED** [review]

**Pull request merged**